crates/stellar-scaffold-macro/src/lib.rs

```diff
@@ -0,0 +1,43 @@
+#![recursion_limit = "128"]
+extern crate proc_macro;
+use proc_macro::TokenStream;
+use std::env;
+
+use quote::quote;
+
+pub(crate) fn manifest() -> std::path::PathBuf {
+    std::path::PathBuf::from(
+        env::var("CARGO_MANIFEST_DIR").expect("failed to finde cargo manifest"),
+    )
+    .join("Cargo.toml")
+}
+
+/// Generates a contract Client for a given contract.
+/// It is expected that the name should be the same as the published contract or a contract in your current workspace.
+///
+/// # Panics
+///
+/// This function may panic in the following situations:
+/// - If `stellar_build::get_target_dir()` fails to retrieve the target directory
+/// - If the input tokens cannot be parsed as a valid identifier
+/// - If the directory path cannot be canonicalized
+/// - If the canonical path cannot be converted to a string
+#[proc_macro]
+pub fn import_contract(tokens: TokenStream) -> TokenStream {
```

Comment:
I think we should rename this as `import_contract_client`. It would be nice to have two macros. One for importing the local wasm. Another would let you reference the name of the deployed contract and then look up the contract id with the registry. Then we could get a client which already knows the contract id too. But definitely have that as a separate PR. That depends on the registry this just relies on the files being in the right place.